omniduct/registry.py

```diff
@@ -82,7 +84,10 @@ def import_from_config(self, config):
             for names, options in config.get(t, {}).items():
                 protocol = options.pop('protocol')
                 register_magics = options.pop('register_magics', True)
-                self.new(names, protocol, register_magics=register_magics, **options)
+                try:
+                    self.new(names, protocol, register_magics=register_magics, **options)
+                except DuctProtocolUnknown as e:
+                    logger.error(str(e))
```

Comment:
I actually liked your more verbose logging from before:

`'Failed to configure `Duct` instance(s) {}. {}'`

Otherwise we won't know which named instances were unable to be configured :).